observer/probers/http/http.go

```diff
@@ -12,11 +13,18 @@ type HTTPProbe struct {
 	url       string
 	rcodes    []int
 	useragent string
+	insecure  bool
 }
 
 // Name returns a string that uniquely identifies the monitor.
+
 func (p HTTPProbe) Name() string {
-	return fmt.Sprintf("%s-%d-%s", p.url, p.rcodes, p.useragent)
+	s := fmt.Sprintf("%s-%d-%s", p.url, p.rcodes, p.useragent)
+	if p.insecure {
+		return s + "-insecure"
+	} else {
+		return s + "-secure"
+	}
```

Comment:
I think it would be safe to return `s + "-insecure" when `p.insecure` is true and `s` otherwise.